localization/english/wc_buildings_l_english.yml

```diff
@@ -917,9 +917,24 @@
  building_type_shaladrassil_desc:0 "A World Tree under which the surviving satyrs from the War of the Ancients were placed in an eternal sleep deep beneath the roots of."
  building_shaladrassil:0 "Shaladrassil"
  building_shaladrassil_desc:0 "A World Tree under which the surviving satyrs from the War of the Ancients were placed in an eternal sleep deep beneath the roots of."
- 
+ building_type_black_rook_hold_01:0 "Black Rook Hold"
+ building_black_rook_hold_01:0 "$building_type_black_rook_hold_01$"
+ building_black_rook_hold_01_desc:0 "Held by the formidable  Lord Kur'talos Ravencrest, Black Rook Hold served as a formidable defense against the Burning Legion in the War of the Ancients. Being forged by ancient elven craftsmen from Val'sharah's towering peak, this nearly invincible fortress is not only the historic residence of Lord Kur'talos, but a commanding center as well."
```

Comment:
what do you mean specifically by commanding center? like a location from which to command an army (command center) or tactical location (commanding position)? if the former, for what army and do they still exist? if the latter, for what operation or battle?